plugins/nsatz/ideal.mli

```diff
@@ -6,6 +6,17 @@
 (*         *       GNU Lesser General Public License Version 2.1        *)
 (************************************************************************)
 
+type metadata = {
+  name_var : string list;
```

Comment:
Are you sure it does not cost anything? While Ocaml 4.04 does actually implement single-field optimization, it has to be manually enabled, as far as I know.